contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -3042,12 +3042,17 @@ ALTER TABLE sys.babelfish_authid_user_ext add COLUMN IF NOT EXISTS user_can_conn
 
 GRANT SELECT ON sys.babelfish_authid_user_ext TO PUBLIC;
 
+-- This is a temporary procedure which is called during upgrade to create guest users
+-- for the user created databases if it doesn't have guest user already.
 CREATE OR REPLACE PROCEDURE sys.babelfish_update_user_catalog_for_guest()
 LANGUAGE C
 AS 'babelfishpg_tsql', 'update_user_catalog_for_guest';
  
 CALL sys.babelfish_update_user_catalog_for_guest();
 
+-- Drop this procedure after it gets executed once.
+DROP PROCEDURE sys.babelfish_update_user_catalog_for_guest();
```

Comment:
Please check with Jungkook/Surendra on whether we want to call drop stmt directly in upgrade script.

Reply:
Then let's put this line at the end of the upgrade script, together with other DROP stmts.
Btw I'm seeing some function is added after resetting the search path, which I think is not right. It's not related to your change, but would you please check with Jungkook if we want to fix that as well within this commit?

Reply:
And please get approval from him as well

Reply:
Ack.